jupyter_server_ydoc/ydoc.py

```diff
@@ -240,6 +242,25 @@ async def clean_room(self) -> None:
         self.websocket_server.delete_room(room=self.room)
 
     def on_document_change(self, event):
+        print("[YDocWebSocketHandler] on_document_change", event)
+        if (
+            isinstance(event, List)
+            and isinstance(event[0], YArrayEvent)
+            and len(event[0].target) == 0
+        ):
+            print("EMPTY:", len(self.room.document._ycells))
+            self.room.document.append_cell(
```

Comment:
`append_cell` is for a notebook, but you didn't check that the document was a notebook.